recipes/folium/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.0" %}
+
+package:
+    name: folium
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/python-visualization/folium.git
+    git_tag: v{{ version }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - jinja2
+        - numpy
+        - pandas
+        - vincent
+        - geopandas
+
+test:
+    imports:
+        - folium
+
+about:
+    home: https://github.com/python-visualization/folium
+    license: MIT License
+    summary: Make beautiful maps with Leaflet.js and Python
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
+        - BibMartin
```

Comment:
Ping @BibMartin

By adding your name here you'll have commit rights in the conda-forge feedstock for folium. Are you OK with that?